### ISSUE TITLE
Tensorflow Keras support of Gradient Accumulation?

### ISSUE THREAD

**Issue by golden0080** [horovod/horovod#2348]
**Is your feature request related to a problem? Please describe.**

No, it's not related to an issue.

**Describe the solution you'd like**

Currently the API supports Gradient Accumulation in Tensorflow and Pytorch module, but not TF.keras.
The proposal is to support GA in tf.keras APIs.

**Describe alternatives you've considered**

Considered to use TF API to do GA, but model was in keras.

**Additional context**

Currently there is no options similar to `backward_passes_per_step` in tf/pytorch.

**Comment by tgaddair** [comment]
Thanks for the feature request @golden0080!  This will be resolved by #2346.

**Comment by golden0080** [comment]
@tgaddair Great to know! I'll take a look at PR and see what I can do there!

**Comment by golden0080** [comment]
I think the PR is pretty much what I requested in this ticket.
